public/companies.json

```diff
@@ -31,5 +31,13 @@
     "location": "Guayaquil, Ecuador",
     "description":
       "Noticias de Ecuador y el mundo en eluniverso.com. Revisa las principales noticias nacionales e internacionales: opinión, política, economía, internacional, deportes, entretenimiento y Guayaquil. Lea las últimas noticias de Ecuador."
+  },
+  {
+    "companyName": "Classy",
+    "industry": "Software Development Agency",
```

Comment:
Mind revising this to be just "Software Development"? The industry should be very generic.